src/meta.js

```diff
@@ -62,7 +62,9 @@ export function mount(microstate, substate, key) {
   }, substate);
 }
 
-export const Profunctor = type(class Profunctor {
+export const Profunctor = type(class {
```

Comment:
I decided to remove the class name from here, too, since the minifier will remove it anyway.